### DISCUSSION TITLE
Welcome to Feeder Discussions!

### DISCUSSION THREAD

**Discussion by spacecowboy** [spacecowboy/Feeder#118] [Announcements]
## 👋 Welcome!
  You can use Discussions as a place to connect with other members of the community. You can:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

And remember

* Please be respectful and helpful to other members of the community and the developer. If you have nothing good to say, maybe say nothing at all.
* Feeder is developed by a single person with a family and full-time job.

**Comment by tiziodcaio** [comment]
Why are issues closed?

**Reply by spacecowboy** [reply]
Issues are closed because people were being too rude and unfriendly.

**Reply by Winson-Huang** [reply]
Then how people report the bugs? Or maybe you mean you do not receive feature requests any more?

**Reply by spacecowboy** [reply]
There is a button inside the menu in the app - "Send a bug report"

This will open your  email client with the correct email address.

You can use the same address for general feedback or feature requests

**Reply by tmplt** [reply]
> You can use the same address for general feedback or feature requests

Adding a note about this in the README would be helpful if issues indeed are a no-go.

**Comment by hrrrps** [comment]
First of, thanks for Feeder, I enjoy it very much. I want to suggest a function, or if it already exists, some instructions for enabling it. I'm not sure what the technical term for this might be, but what I'm thinking of is something like a 'furthest page read' function so that if you stop reading an article you can come back to it and continue reading from where you left off. This would be especially helpful for the longer form articles that are more difficult to finish in one sitting.

**Reply by spacecowboy** [reply]
Please make a new topic of type `Idea` for suggestions.
